controllers/metal3machine_controller_test.go

```diff
@@ -439,4 +440,56 @@ var _ = Describe("Metal3Machine manager", func() {
 		),
 	)
 
+	type TestCaseClusterToBMM struct {
+		Cluster       *capi.Cluster
+		Machine       *capi.Machine
+		ExpectRequest bool
+	}
+
+	DescribeTable("Cluster To Metal3Machines tests",
+		func(tc TestCaseClusterToBMM) {
+			objects := []runtime.Object{
+				tc.Cluster,
+				tc.Machine,
+			}
+			c := fake.NewFakeClientWithScheme(setupScheme(), objects...)
+			r := Metal3MachineReconciler{
+				Client: c,
+			}
+			obj := handler.MapObject{
+				Object: tc.Cluster,
+			}
+			reqs := r.ClusterToMetal3Machines(obj)
+
+			if tc.ExpectRequest {
+				Expect(len(reqs)).To(Equal(1), "Expected 1 request, found %d", len(reqs))
```

Comment:
you should add a test case with multiple machines

Reply:
I was following the existing workflow, tests for `BareMetalHost To Metal3Machines` also does not have any test with multiple machines. I can add it anyhow.

Reply:
yes, that would be better

Reply:
Added